prusti-interface/src/utils.rs

```diff
@@ -210,22 +210,24 @@ pub fn get_attributes(tcx: ty::TyCtxt<'_>, def_id: DefId) -> &[prusti_rustc_inte
 /// Any arguments of the attribute are ignored.
 pub fn has_prusti_attr(attrs: &[ast::Attribute], name: &str) -> bool {
     attrs.iter().any(|attr| match &attr.kind {
-        ast::AttrKind::Normal(
-            ast::AttrItem {
-                path:
-                    ast::Path {
-                        span: _,
-                        segments,
-                        tokens: _,
-                    },
-                args: _,
-                tokens: _,
-            },
-            _,
-        ) => {
-            segments.len() == 2
-                && segments[0].ident.as_str() == "prusti"
-                && segments[1].ident.as_str() == name
+        ast::AttrKind::Normal(normal_attr) => {
+            match &normal_attr.item {
+                ast::AttrItem {
+                    path:
+                        ast::Path {
+                            span: _,
+                            segments,
+                            tokens: _,
+                        },
+                    args: ast::MacArgs::Empty,
```

Comment:
@vakaras This broke it, some of our attributes are `::Eq` :)

I'll push the fix + update to today's toolchain.

Reply:
Oh, thank for finding this out!